packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/internal/AppConfigurationImpl.kt

```diff
@@ -39,8 +37,9 @@ import io.realm.kotlin.mongodb.HttpLogObfuscator
 import org.mongodb.kbson.ExperimentalKBsonSerializerApi
 import org.mongodb.kbson.serialization.EJson
 
-// TODO Public due to being a transitive dependency to AppImpl
+internal const val MISSING_BUNDLE_ID = "UNKNOWN_BUNDLE_ID"
```

Comment:
👍 It was referenced multiple times before I realized I had to rewire `App.create` through the compiler plugin too. Now it is just hardcoded, since only used once.

Reply:
But now the hardcoded in https://github.com/realm/realm-kotlin/pull/1411/files#diff-fe5174d6d660c1d744420ab76fec40865cd650335735ff3a6dc8f41f4bf5163d ... is this not OK 🙈

Reply:
Should we move any constants that are lingering around the library into a single constants file?